src/A2C_model.py

```diff
@@ -13,25 +13,30 @@ def call(self, logits, **kwargs):
 class Model(Model):
     def __init__(self, ac_dim):
         super(Model, self).__init__()
-        self.conv1 = Conv2D(16, 3, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv2 = Conv2D(16, 3, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
-        self.conv3 = Conv2D(32, 3, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv1 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
+        self.conv2 = Conv2D(16, 2, activation='relu', kernel_regularizer=tf.keras.regularizers.l2(1e-4))
```

Comment:
Consider passing these numbers as arguments to your class. It will reduce/eliminate the need to make changes like this later. But don't overthink it. If you can move faster the way you're currently doing it, then keep doing it and we'll worry about this later